src/hevm/src/EVM/Solidity.hs

```diff
@@ -218,7 +218,7 @@ readSolc fp =
       Nothing -> return Nothing
       Just (contracts, asts, sources) -> do
         sourceCache <- makeSourceCache sources asts
-        return $! Just (contracts, sourceCache)
+        return (Just (contracts, sourceCache))
```

Comment:
hmm, the `$!` also imposes strictness so not sure this is just a syntactic change

Reply:
nice catch, added it back.